test/Tester/SerializationTests/SerializationTests.DifferentTypes.cs

```diff
@@ -123,16 +122,18 @@ public static void Register()
             }
         }
 
+#if !NETSTANDARD_TODO
         [Fact, TestCategory("BVT"), TestCategory("Functional"), TestCategory("Serialization")]
         public void SerializationTests_RecursiveSerialization()
         {
             TestTypeA input = new TestTypeA();
             input.Collection = new HashSet<TestTypeA>();
             input.Collection.Add(input);
 
-            TestTypeA output1 = TestingUtils.RoundTripDotNetSerializer(input);
+            TestTypeA output1 = Orleans.TestingHost.Utils.TestingUtils.RoundTripDotNetSerializer(input);
```

Comment:
I do not know, and really didn't care for this PR, since I was just setting up the plumbing needed by @xiazen to start adding tests. I'll defer this comment for her upcoming PR